applications/GeoMechanicsApplication/tests/test_normal_load_on_hexa_element.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+
+class KratosGeoMechanicsNormalLoadHexaTests(KratosUnittest.TestCase):
+    """
+    This class contains tests for applying normal loads on 3D elements.
+    """
+
+    def test_hexa_8n_normal_load(self):
+        test_name = 'hexa_8n_normal_load'
+        file_path = test_helper.get_file_path(test_name)
+        test_helper.run_kratos(file_path)
+
+        all_integration_points = [0, 1, 2, 3, 4, 5, 6, 7]
+        output_file_path = os.path.join(file_path, test_name + '.post.res')
+        output_reader = test_helper.GiDOutputFileReader()
+        output_data = output_reader.read_output_from(output_file_path)
+        total_stresses = \
```

Comment:
`total_stresses` => `total_stress_vectors`?

Reply:
Agreed, is more clear!